lib/events/dynamoevents/dynamoevents.go

```diff
@@ -503,11 +507,11 @@ func (l *Log) SearchEvents(fromUTC, toUTC time.Time, filter string, limit int) (
 	var lastEvaluatedKey map[string]*dynamodb.AttributeValue
 	var total int
 
-	// We iterate over the responses until we roughly hit 100MB in size. This is an arbitrary limit to prevent
-	// runaway loops.
-	// DynamoDB reponses are capped at 900KB according to their documentation.
-	const maxResponseSize = 100 * 1024
-	for responseSize := 0; responseSize < maxResponseSize; responseSize += 900 {
+	// We iterate over the responses until we roughly hit ~90MB in size.
+	// This is an arbitrary limit to prevent runaway loops.
+	const maxResponseSizeKbs = maxPageSizeKbs * 100
+
+	for responseSize := 0; responseSize < maxResponseSizeKbs; responseSize += maxPageSizeKbs {
```

Comment:
And it's not just going from `0` to `99` because it's more descriptive that way? Sorry if I missed that comment.

Reply:
Yeah, this is basically `for i := 0; i < 100; i++` with extra steps.
You can just leave a comment saying that 100 maxed-out responses from dynamo here will eat ~100MB